opmd_viewer/openpmd_timeseries/interactive.py

```diff
@@ -40,6 +43,11 @@ def slider(self, figsize=(6, 5),
         kw: dict
             Extra arguments to pass to matplotlib's imshow
         """
+        # Check that the dependencies have been installed
+        if not dependencies_installed:
```

Comment:
instead of defining a global one could also just try to access, e.g. `widgets` here and re-through the own error if it fails.